sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Cast.scala

```diff
@@ -204,12 +206,17 @@ object Cast {
       > SELECT _FUNC_('10' as int);
        10
   """)
-case class Cast(child: Expression, dataType: DataType, timeZoneId: Option[String] = None)
+case class Cast(
+    child: Expression,
+    dataType: DataType,
+    timeZoneId: Option[String] = None,
+    format: Option[String] = None)
   extends UnaryExpression with TimeZoneAwareExpression with NullIntolerant {
 
   def this(child: Expression, dataType: DataType) = this(child, dataType, None)
 
-  override def toString: String = s"cast($child as ${dataType.simpleString})"
+  override def toString: String =
+    s"cast($child as ${dataType.simpleString}${format.map(" FORMAT " + _).getOrElse("")})"
```

Comment:
nit: Make `FORMAT` in lower case as `cast` and `as`

Reply:
fixed